class NotFoundException(Exception):
    pass


class BadResponseException(Exception):
    pass
